packages/vue/src/sdk.ts

```diff
@@ -61,6 +61,22 @@ Update your \`Sentry.init\` call with an appropriate config option:
   }
 }
 
+/**
+ * Initialize Vue-specific error monitoring for a given Vue app.
+ */
+export function initVueApp(app: Vue): void {
```

Comment:
I wonder if we should pass in client as an optional arg.

Reply:
sure, why not! will adjust it

Reply:
I added an optional `client` argument to be passed.